flow360/component/case.py

```diff
@@ -312,7 +312,7 @@ def submit(self, force_submit: bool = False) -> Case:
         data = {
             "name": self.name,
             "meshId": volume_mesh_id,
-            "runtimeParams": self.params.flow360_json(),
+            "runtimeParams": self.params.json(),
```

Comment:
I think we need to tie this python (develop branch) to one version of solver so that we can still use develop branch to submit cases. Is solver ready?

Reply:
Good question. I have a PR for Core listed above. I am still working on testcases PR which changed quite a bit because of this.